### DISCUSSION TITLE
Zettelkasten file names in Obsidian

### DISCUSSION THREAD

**Discussion by rdataman2000** [BryanJenksCommunity/FAQ#26] [Q&A] [answered]
Hello, I have a question about Zettelkasten file names in Obsidian. I am trying to determine whether to use a Zettelkasten prefix. I do like the idea of being able to sort my notes chronologically.
1) first of all, is there real benefit in date/time stamps?
2) are there any drawbacks to using them? 
3) If i do go with it, does it really matter whether i use the default 202012101322 or something custom like 2020-12-10_1322 <specific note title> or even 2020-12-10_1322Thu<specific note title>?
4) If i go with something custom, and then decide later that i would like to completely change my naming format, could i just use regex/grep to change all occurrences of names (including in links)?
5) OR, might it be better to use Luhman's system: 1, 1a, 1a1,1a2,1b1, etc?
6) does any of this make linking notes more complicated, or make links more prone to breaking?
7) how's about just make notes with descriptive titles and call it a day.
Any recommendations are welcome. Thanks!

**Comment by tallguyjenks** [comment, marked as answer]
I personally do not find any use for the prefixer at all. Want to sort chronologically? you can do that in Obsidian already by either modified or created date/time so that already covers your use case. Plus the prefixer makes it annoying to deal with your note names because to make "pretty" names you'll need to copiously use aliases.

1. i don't really see one anymore for digital zettelkasten's
2. see above paragraph
3. the formats you listed should be fine
4. probably, but it might be a little messy
5. again, that system was used because it was an analog system and didn't have the benefits of a digital system that we have so again i find it unecessary
6. with obsidian you have the option to refactor note names if you change one within the app across the whole vault, the only really annoying this is getting "pretty" note names without a 12-14 serial number in the title requiring aliases to get a pretty display name 
7. that's my approach

**Reply by rdataman2000** [reply]
Man, thanks, extremely helpful. Puts my mind at ease. I will stick with option 7. BTW, your youtube vids are great, very informative. Thanks again!

**Reply by tallguyjenks** [reply]
Glad it helps :)and that you're enjoying the videos :)

**Reply by Protektorius** [reply]
I second that, thanks @tallguyjenks for taking the time, this was very helpful! I better go check out those videos too.... ;)

**Reply by mikkovedru** [reply]
@tallguyjenks 
In your "My 2021 Comprehensive Obsidian Zettelkasten Workflow" video you said that you are using dates in the title names (1:22:31, https://www.youtube.com/watch?v=wB89lJs5A3s&t=4951s ).

Did your opinion change in this regard compared to the one you wrote in January?

Could you please explain the logic behind:
1. Why are you putting dates only after you processed the note and not when you first read it or put it in your system? You are using statuses in order to see if the note needs any more work, so you don't need the fact of a date missing in the title for that. But sometimes the processing might take a long time, so wouldn't it be more convenient to have the date closer to the date you first encountered it (or the date that article was published), instead of when you finally processed it (which is a-difficult-to-define-thing anyway)?
2. Why some of your finished notes/entries don't have dates, while the others do?

Thank you for the great videos!

**Reply by tallguyjenks** [reply]
@mikkovedru Nope, its not for a Zettelkasten prefixer to uniquely identify a note, it's the date stamp of when I finished processing that input for usage in my timeline note so I can see the timeline of inputs I've consumed.

if no date, haven't completed processing the note :)

timeline note for reference: 

![image](https://user-images.githubusercontent.com/29872822/133913975-05c66510-421a-4eec-8c98-e5a18ab97f2d.png)

**Reply by mikkovedru** [reply]
@tallguyjenks 
Thank you for your answer and for the reference picture.

I see that you have the finished notes that still have a red status. According to your video, red status = "Not Processed". Could you explain please?

FYI Supercharged Links plugin now supports the file tree, which means that you can do all kinds of stuff using the frontmatter values instead of only relying on the first character of the file name. 
![image](https://user-images.githubusercontent.com/2521942/138770642-3460f9ff-ca87-44ca-bf0c-74986e257af7.png)

**Reply by tallguyjenks** [reply]
very cool on the file tree, ill have to look into that, the special symbol serves multiple purposes so this wont make it go away for me though :)

**Reply by reissmann** [reply]
I have one question on this: 
In your awesome 4h video, it looks like you only have added the tags on the timeline, but not when you just link from one note to another. I am able to add the tags using supercharged links, but they are then enabled everywhere (and all notes that do not have tags are then named "null Title". Any tips for me? :)

**Reply by tallguyjenks** [reply]
@reissmann look at the supercharged links settings potentially, I'm a visual person so if that's not it I'll need examples i can see

**Reply by reissmann** [reply]
Thank you for the fast reply. I'm sorry, I'll provide some more info :)

I added this css to get the tags-icons added to my links:
````
a.internal-link[data-link-tags]::before{
    content: attr(data-link-tags)" "
}
````

But, now the tags are added everywhere, not only on the timeline.

![screenshot-1643305010](https://user-images.githubusercontent.com/833161/151413182-d7651992-4837-4bc3-b8be-e0d1271a3928.png)

So, I guess you are usaing a specific css modifier? Or add a specific `csstag`to your frontmatter for only the timeline? Or there is a supercharged links attribute that I just don't spot? :)

**Reply by tallguyjenks** [reply]
@reissmann in obsidian's plugin settings you can determine where supercharged links displays:

![image](https://user-images.githubusercontent.com/29872822/151428178-ec16af1e-3afc-40c1-b61f-dbfc517fd760.png)

**Reply by reissmann** [reply]
I think I just completely missed that you were in edit mode all time in your video, while I'm using preview mode a lot when editing. 
Thanks a lot for clarification and for your time!
Best regards.

**Reply by tallguyjenks** [reply]
@reissmann  yup! im rarely in preview mode, however i do use live preview now :)
